osh/client/commands/cmd_mock_build.py

```diff
@@ -15,6 +16,16 @@ def options(self):
 
         # this option cannot be used for diff-build tasks
```

Comment:
Should we need to change the comment?

Reply:
Not sure I understand. The changes in this MR doesn't change the behavior of `diff-build`, I think it's still true that the option cannot be used for diff-build.

Reply:
I think `diff-build` would not work with `cspodman` anyway.  It should probably work with `version-diff-build` though.

Reply:
Latested updates applied both client/hub side validation.